WORKSPACE

```diff
@@ -73,7 +73,7 @@ maven_install(
     ],
 )
 
-load("@maven//:defs.bzl", "pinned_maven_install")
+load("@protobuf_maven//:defs.bzl", "pinned_maven_install")
```

Comment:
Is this change correct? In the error logs for the failing tests we see:

```
ERROR: Failed to load Starlark extension '@@protobuf_maven//:defs.bzl'.
Cycle in the workspace file detected. This indicates that a repository is used prior to being defined.
The following chain of repository dependencies lead to the missing definition.
```

Reply:
I think the problem was I forgot to add the `name = "protobuf_maven"` in the WORKSPACE. I only did it in MODULE.bazel for the first commit.